config/webpack.config.js

```diff
@@ -43,22 +43,12 @@ module.exports = {
 							// TODO: Look into specifying our minimum versions in preset-env
 							// for enhanced experience instead of making everything become ES5
 							presets: [
-								require.resolve('babel-preset-es3'),
-								require.resolve('babel-preset-env')
+								require.resolve('@babel/preset-env')
```

Comment:
Do we need to specify [targets](https://babeljs.io/docs/en/babel-preset-env#targets) now?

Reply:
>if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.

I guess we could use a target of:
```{
  "targets": {
    "ie": "11"
  }
}
```

Reply:
👍